talk/objectorientation/objectsclasses.tex

```diff
@@ -92,7 +92,7 @@
     \begin{itemize}
     \item usually in .cpp, outside of class declaration
     \item using the class name as namespace
-    \item when reference to the object is needed, use {\it this} keyword
+    \item when reference to the object is needed, use \mintinline{cpp}{this} keyword
```

Comment:
Shouldn't this be: `when a pointer to the object ...`?

Reply:
Both, actually, isn't it? But I'll change it to pointer.

Reply:
I used both now.